### PR TITLE
Fix comments and names

### PR DESCRIPTION
**Pull request by tyomitch**
No functional change

### CODE REVIEW

**Comment by tyomitch** [comment]
> Apparently you have way more time to dive into this than me. 😃

Got to blame the covid lockdown for that!

File: src/renderer/renderer.es6 [hunk 1]
```diff
@@ -201,7 +201,7 @@ export default function Renderer(phonemes, pitch, mouth, throat, speed, singmode
       }
       // unvoiced
       off = pitch ^ 255 & 0xFF; // unsigned char
-      let mem53 = tab48426[hibyte] & 0xFF; // unsigned char
+      let value0 = sampledConsonantValues0[kind] & 0xFF; // unsigned char
```
Comment (discordier) [inline src/renderer/renderer.es6:204]: When renaming `mem53` to `value0` and `hibyte` to `kind`, please pay attention to do this on all occasions.

I have amended this PR on my side now in c56978571af76a22210926e202d74f17b17518c5.

**Comment by discordier** [comment]
Merged in 01c8511, will be part of 0.1.3.

File: src/renderer/renderer.es6 (hunk 1 above)
Comment (tyomitch) [reply src/renderer/renderer.es6:204]: Sorry, those names remained from a bigger patch I had locally -- now submitted as https://github.com/discordier/sam/pull/14

**Pull request merged**